docs/application/web/guides/multimedia/metadata.md

```diff
@@ -23,27 +42,126 @@ Additionally, to access files using the Metadata API (in [mobile](../../api/late
 
+## Extract artwork
+
+1. Create a file handle using `createFileHandle()`. Ensure your file includes an artwork.
+
+   ```javascript
+   var filePath = "music/sample.mp3";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is created, artwork can be extracted:
+
+   ```javascript
+   var artwork = fileHandle.getArtwork();
+   ```
+
+3. Blob object can be easily shown in your application in html *&lt;img&gt;* tag
+
+   ```javascript
+   var elem = document.getElementById("blobImage");
+   elem.src = URL.createObjectURL(artwork);
+   ```
+
+4. After use, release the file handle.
+
+## Extract thumbnail frame
+
+1. Create a file handle using `createFileHandle()` for a video file.
+
+   ```javascript
+   var filePath = "videos/sample_video.mp4";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is created, thumbnail frame can be extracted:
+
+   ```javascript
+   var thumbnail = fileHandle.getThumbnailFrame();
+   ```
+
+3. Blob object can be easily shown in your application in html *&lt;img&gt;* tag
+
+   ```javascript
+   var elem = document.getElementById("blobImage");
+   elem.src = URL.createObjectURL(thumbnail);
+   ```
+
+4. After use, release the file handle.
+
+## Extract certain frame
+
+1. Create a file handle using `createFileHandle()` for a video file.
+
+   ```javascript
+   var filePath = "videos/sample_video.mp4";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is created, thumbnail frame can be extracted. You need to provide timestamp information (in milliseconds) and a flag indicating accuracy. When the flag value is set to 'true', flag indicates extracting an exact frame for the given time. When the value is 'false', flag indicates extracting an <a href="https://en.wikipedia.org/wiki/Video_compression_picture_types">I-frame</a> nearest to the given time. Gathering nearest I-frame has better performance.
+
+   ```javascript
+   var timestamp = 2000;
+   var returnExactFrame = true;
+   var frame = fileHandle.getFrameAtTime(timestamp, returnExactFrame);
+   ```
+
+3. Blob object can be easily shown in your application in html *&lt;img&gt;* tag
+
+   ```javascript
+   var elem = document.getElementById("blobImage");
+   elem.src = URL.createObjectURL(frame);
+   ```
+
+4. After use, release the file handle.
+
+## Extract lyrics
+
+1. Create a file handle using `createFileHandle()`. Lyrics need to be included in multimedia file as SYLT metadata tag. Supported formats of encoding are at least <a href="http://www.ietf.org/rfc/rfc2279.txt">UTF-8</a> and <a href="http://en.wikipedia.org/wiki/ISO/IEC_8859-1">ISO-8859-1</a>. Other formats can be ignored depending on the platform.
+
+   ```javascript
+   var filePath = "music/sample.mp3";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is created, one need to check how many lyrics are included.
```

Comment:
In my previous comment, I had removed "one need to" as it is implicit that the developer needs to check this. Secondly, in technical documentation, we use second-person pronouns such you, your, etc., and avoid using third-person pronouns completely. 

Is there any reason you retained it?
If it is was an oversight could you change to:

2. When the file handle is created, check how many lyrics are included.

Reply:
Yes, I've missed this part. Sometimes it is easy to miss some changes in sentence (especially, if there are many comments). Maybe you should consider using github built-in 'suggestion' mechanism in your comments - it is much easier then to apply a suggestion with just one click without any manual copy-paste.